frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.unit.spec.tsx

```diff
@@ -169,6 +170,25 @@ describe("TagEditorParam", () => {
       });
     });
 
+    it("should default to number/= for a new reviews->rating field filter (metabase#16151)", async () => {
```

Comment:
This is essentially a duplicate of the test above *should default to number/= for a new numeric field filter*, with a different field selected to match the reported issue.